DashWallet/Sources/UI/DashPay/Items/Objects/Notifications/DWDPNewIncomingRequestNotificationObject.h

```diff
@@ -22,7 +22,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface DWDPNewIncomingRequestNotificationObject : DWDPNewIncomingRequestObject <DWDPNotificationItem>
 
-- (instancetype)initWithBlockchainIdentity:(DSBlockchainIdentity *)blockchainIdentity NS_UNAVAILABLE;
+- (instancetype)initWithBlockchainIdentity:(DSBlockchainIdentity *)blockchainIdentity; // if !MOCK_DASHPAY NS_UNAVAILABLE;
```

Comment:
It's difficult to create a valid `DSFriendRequestEntity`, so this method is the easy way to mock a notification.
When the Platform is restored, we will search and clean up all of the `MOCK_DASHPAY`-related changes.